app/src/main/kotlin/edu/stanford/bdh/engagehf/health/symptoms/SymptomsViewModel.kt

```diff
@@ -56,7 +58,16 @@ class SymptomsViewModel @Inject internal constructor(
 
     fun onAction(action: Action) {
         when (action) {
-            Action.Info -> {}
+            is Action.Info -> {
+                val selectedSymptomType =
```

Comment:
This check is not needed, we are only rendering the icon in case of success state, directly emitting `appScreenEvents.emit(AppScreenEvents.Event.SymptomsDescriptionBottomSheet)` should be enough